man/zram-generator.conf.md

```diff
@@ -80,6 +80,12 @@ Devices with the final size of *0* will be discarded.
 
   If unset, none will be configured and the kernel's default will be used.
 
+* `swap-priority`=
+
+  Controls the relative swap priority, a value between -1 and 32767. Higher numbers indicate higher priority.
```

Comment:
Isn't the range full i32?

Reply:
I took the range from swapon(8). I think the negative values are not supposed to be configured directly, but are used by the kernel.